src/test/java/com/wikia/webdriver/testcases/adstests/TestUrlBuilder.java

```diff
@@ -1,378 +1,420 @@
-                    "http://leagueoflegends.sandbox-mercurydev.wikia.com/wiki/League_of_Legends_Wiki",
-                    "http://leagueoflegends.dmytror.wikia-dev.pl/wiki/League_of_Legends_Wiki",
-                    "http://leagueoflegends.wikia-staging.com/wiki/League_of_Legends_Wiki"
-            },
-            {
-                    "wowwiki", "wiki/Portal:Main", true, true,
-                    "https://wowwiki.wikia.com/wiki/Portal:Main",
-                    "https://wowwiki.preview.wikia.com/wiki/Portal:Main",
-                    "https://wowwiki.sandbox.wikia.com/wiki/Portal:Main",
-                    "https://wowwiki.sandbox-mercurydev.wikia.com/wiki/Portal:Main",
-                    "https://wowwiki.dmytror.wikia-dev.pl/wiki/Portal:Main",
-                    "https://wowwiki.wikia-staging.com/wiki/Portal:Main"
-            },
-    };
+  private static Object[][] TEST_DATA = new Object[][]{
+      {
+          "runescape", "wiki/RuneScape_Wiki", false, false,
+          "http://runescape.wikia.com/wiki/RuneScape_Wiki",
+          "http://runescape.preview.wikia.com/wiki/RuneScape_Wiki",
+          "http://runescape.sandbox.wikia.com/wiki/RuneScape_Wiki",
+          "http://runescape.sandbox-mercurydev.wikia.com/wiki/RuneScape_Wiki",
+          "http://runescape.dmytror.wikia-dev.pl/wiki/RuneScape_Wiki",
+          "http://runescape.wikia-staging.com/wiki/RuneScape_Wiki"
+      },
+      {
+          "yugioh", "wiki/Main_Page", false, false,
+          "http://yugioh.wikia.com/wiki/Main_Page",
+          "http://yugioh.preview.wikia.com/wiki/Main_Page",
+          "http://yugioh.sandbox.wikia.com/wiki/Main_Page",
+          "http://yugioh.sandbox-mercurydev.wikia.com/wiki/Main_Page",
+          "http://yugioh.dmytror.wikia-dev.pl/wiki/Main_Page",
+          "http://yugioh.wikia-staging.com/wiki/Main_Page"
+      },
+      {
+          "naruto", "wiki/Narutopedia", false, false,
+          "http://naruto.wikia.com/wiki/Narutopedia",
+          "http://naruto.preview.wikia.com/wiki/Narutopedia",
+          "http://naruto.sandbox.wikia.com/wiki/Narutopedia",
+          "http://naruto.sandbox-mercurydev.wikia.com/wiki/Narutopedia",
+          "http://naruto.dmytror.wikia-dev.pl/wiki/Narutopedia",
+          "http://naruto.wikia-staging.com/wiki/Narutopedia"
+      },
+      {
+          "leagueoflegends", "wiki/League_of_Legends_Wiki", false, false,
+          "http://leagueoflegends.wikia.com/wiki/League_of_Legends_Wiki",
+          "http://leagueoflegends.preview.wikia.com/wiki/League_of_Legends_Wiki",
+          "http://leagueoflegends.sandbox.wikia.com/wiki/League_of_Legends_Wiki",
+          "http://leagueoflegends.sandbox-mercurydev.wikia.com/wiki/League_of_Legends_Wiki",
+          "http://leagueoflegends.dmytror.wikia-dev.pl/wiki/League_of_Legends_Wiki",
+          "http://leagueoflegends.wikia-staging.com/wiki/League_of_Legends_Wiki"
+      },
+      {
+          "wowwiki", "wiki/Portal:Main", true, false,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "wiki/Portal:Main" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)